lemur/notifications/messaging.py

```diff
@@ -30,7 +30,7 @@
 
 def get_certificates(exclude=None):
     """
-    Finds all certificates that are eligible for notifications.
+    Finds all certificates that are eligible for expiration notifications.
```

Comment:
One thought about the query in this method, we should [add filter](https://github.com/Netflix/lemur/blob/master/lemur/notifications/messaging.py#L44) for `Certificate.revoked == True` as well.

Reply:
Makes sense, no reason to send any notifications for revoked certs.